gateways/istio-ingress/kustomization.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  - istio-ingress.yaml
```

Comment:
This doesn't seem to be a file that exists?

Reply:
fixed - I still don't know if checking in generated files is right, but golang gen files and vendor are doing it so we may be ok for 1.2

Reply:
I think its ok, as long as its deterministic. There was some stuff different when I ran it vs yours - some stuff has `costin` in it at the very least and I think some may not have consistent ordering.